### PR TITLE
Location Modal Style Update

### PR DESCRIPTION
**Pull request by MrT3313**
# Description

Please include a summary of the change and a link to which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status
- [ x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ x] Test A
- [ ] Test B

# Checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ x] There are no merge conflicts

### CODE REVIEW

File: client/src/components/Review/AllReviewsPanel.js [hunk 1]
```diff
@@ -1,95 +1,72 @@
...
+  // RENDER
+  render() {
+    return (
+      <StyleModal>
+        <Header> Reviews </Header>
+        <Content>      
+          <ul className='ratingInfo'>
             { this.state.reviews.map(review =>
-            <li>UserId: {review.user_id},  Rating: {review.rating}, Comments:{review.comments}</li>)}
+              <li>
```
Comment (CLantigua2) [inline client/src/components/Review/AllReviewsPanel.js]: This li tag will require a key or this can cause rendering issues down the line. Best to take care of this early to avoid weird bugs.

File: client/src/components/Review/DetailsPanel.js [hunk 2]
```diff
@@ -32,42 +86,64 @@ class DetailsPanel extends React.Component {
...
-                  <p>Comments: {location.comment}</p>{" "}
-                </li>
-              </ul>
-            );
-          })}
+            <p>{this.props.details[1]}</p>
+          <h2 className='hours'>Hours:</h2>
+          <ul>
+            {this.props.hours.map(day => {
+              return <li><div>{day}</div></li>;
```
Comment (CLantigua2) [inline client/src/components/Review/DetailsPanel.js]: also needs a key, would also restructure this to make sense, I don't see how this.props.hours is expected to return a day

File: client/src/components/Review/DetailsPanel.js [hunk 3]
```diff
@@ -32,42 +86,64 @@ class DetailsPanel extends React.Component {
...
+            <STYLED_featuredReview>
+              Latest Review
+            </STYLED_featuredReview>
+            {this.state.location_id.map(location => {
+              return (
+                <ul key={location.id}>
+                  {/* <li>
+                    {" "}
+                    <p>UserId: {location.user_id},</p>{" "}
+                  </li> */}
```
Comment (CLantigua2) [inline client/src/components/Review/DetailsPanel.js]: Why is this here?

Comment (MrT3313) [reply client/src/components/Review/DetailsPanel.js]: its related to my slack comment about if we are going to use the userID in our review process. In this case I commented it out but if you look on the actual add a review I left it in there until we decide what we want to do.
